### PR TITLE
chore: include example resource requests

### LINKED ISSUE #103: Update k8s examples to include example resource requests
Hi,

are there any cpu and memory limits recommendations for kubernetes usage?

That could be worth mentioning in the README.md.

### PR DESCRIPTION
**Pull request by enocom**
Fixes #103.

### CODE REVIEW

File: examples/kubernetes/proxy_with_sa_key.yaml [hunk 1]
```diff
@@ -57,6 +57,20 @@ spec:
           mountPath: /secrets/
           readOnly: true
           # [END cloud_sql_proxy_k8s_volume_mount]
+        # Resource configuration depends on an application's requirements. The
+        # number of connections is correlated to memory use. The amount of IO
+        # across each connection is correlated to CPU use. Users should adjust
+        # the following values based on what their application needs.
+        # For details, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
```
Comment (kurtisvg) [inline examples/kubernetes/proxy_with_sa_key.yaml:64]: 1. Let's avoid stating the same thing twice
2. "Users" -> "You" to match the rest of the comments

```suggestion
        # Resource configuration depends on an application's requirements.  You
        # should adjust the following values based on what their application needs.
        # For details, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
```

File: examples/kubernetes/proxy_with_sa_key.yaml [hunk 2]
```diff
@@ -57,6 +57,20 @@ spec:
...
+        resources:
+          requests:
+            # More connections require more memory. Fewer connections require
```
Comment (kurtisvg) [inline examples/kubernetes/proxy_with_sa_key.yaml:67]: Generally we prefer active over passive: 

```
The proxy's memory use scales linearly with the number of active connections. Fewer open connections 
will use less memory. Adjust this value based on your applications's requirements. 
```

File: examples/kubernetes/proxy_with_sa_key.yaml [hunk 3]
```diff
@@ -57,6 +57,20 @@ spec:
...
+            # less memory. Adjust this value based on your application's
+            # requirements.
+            memory: "2Gi"
+            # More database IO require more CPUs. Less database IO require less
```
Comment (kurtisvg) [inline examples/kubernetes/proxy_with_sa_key.yaml:71]: Same here as above.

**Review by kurtisvg — APPROVED** [review]

File: examples/kubernetes/proxy_with_sa_key.yaml (hunk 1 above)
Comment (enocom) [reply examples/kubernetes/proxy_with_sa_key.yaml]: Fixed

File: examples/kubernetes/proxy_with_sa_key.yaml (hunk 2 above)
Comment (enocom) [reply examples/kubernetes/proxy_with_sa_key.yaml]: Fixed.

File: examples/kubernetes/proxy_with_sa_key.yaml (hunk 3 above)
Comment (enocom) [reply examples/kubernetes/proxy_with_sa_key.yaml]: Fixed.

**Pull request merged**